index.html

```diff
@@ -1,9 +1,20 @@
 <!DOCTYPE html>
 <html>
 <head>
```

Comment:
You're head and body tags should be indented once from the html tags because they are contained inside of them. (I'm not sure why atom doesn't correct this)